kahuna/public/js/upload/manager.js

```diff
@@ -31,7 +31,21 @@ upload.factory('uploadManager',
         };
     }
 
-    async function createJobItems(files){
+    async function createJobItems(_files){
+
+      const maybeUploadLimitInBytes = window._clientConfig.maybeUploadLimitInBytes;
+      const maybeFilesAboveSizeLimit = maybeUploadLimitInBytes && _files.filter(file => file.size > maybeUploadLimitInBytes);
```

Comment:
I think the type of `maybeFilesAboveSizeLimit` ends up being `0 | undefined | File[]`, which should work fine in practice (assuming that we'd never want to actually enforce an upload limit of 0 bytes), but feels a bit awkward. Might be easier to reason about if we explicitly turned `0 | undefined` to `false`..?

```suggestion
      const maybeFilesAboveSizeLimit = !!maybeUploadLimitInBytes && _files.filter(file => file.size > maybeUploadLimitInBytes);
```

Reply:
not blocking, though